src/plugins/synced-lyrics/renderer/lyrics/progress.ts

```diff
@@ -16,20 +17,21 @@ export const resetAllVariables = () => {
 
 export const createProgressEvents = (on: Function) => {
   on('synced-lyrics:paused', (data: PlayPauseEvent) => {
-      if (data.isPaused) 
+      if (data.isPaused)
           clearInterval(interval!);
   });
 
   on('synced-lyrics:setTime', (t: number) => {
     if(!lyrics && !songWithLyrics) return;
     if (config.preciseTiming) {
-        currentTime = secToMilisec(t);
-        clearInterval(interval!);
-        interval = setInterval(() => {
-          currentTime += 10;
-          changeActualLyric(currentTime);
-        }, 10);
-    } 
+      // MAGIC: Since the transition takes 300ms, we need to add a delay of 300ms to the current time
+      currentTime = secToMilisec(t) + 300;
```

Comment:
Also, since this change, the lyrics seems to be displayed too early now. But if the way it worked before would have been a source of error we can keep it like that